dnf/plugin.py

```diff
@@ -199,13 +199,34 @@ def _import_modules(package, py_files):
 
 
 def _iter_py_files(paths, skips, enable_plugins):
+    plugins = []
+    skips = set(skips)
+    enable_plugins = set(enable_plugins)
+    pattern_found = set()
     for p in paths:
         for fn in glob.glob('%s/*.py' % p):
-            (name, _) = os.path.splitext(os.path.basename(fn))
-            if any(fnmatch.fnmatch(name, pattern) for pattern in skips):
-                if not any(fnmatch.fnmatch(name, pattern) for pattern in enable_plugins):
-                    continue
-            yield fn
+            (name, base_path) = os.path.splitext(os.path.basename(fn))
```

Comment:
Sorry about piecemealing this :disappointed:, but the name `base_path` here isn't good, its actually the file extension. And its a throwaway variable, I assume you changed it because it conflicts with the i18n translation function `_()`? How about using `__` for throwaway variables, which seems what people are using now to avoid the conflict, at least according to my brief google search?

Reply:
@lukash Renamed to usused

Reply:
I think I like that even less than naming it according to what it is. Underscore(s) are a convention and everyone knows what it is. This just looks weird...

Also, when I suggest a change and you do something else, I kind of expect you to provide some reasoning... And ideally to have the discussion before you actually do it. When I say: _"Could you do A?"_ and you say: _"I did B."_ I really don't know what to think...